src/pricing/cc-pricing-product.smart-addon.js

```diff
@@ -33,9 +33,17 @@ defineComponent({
       }),
 
       context$.subscribe(({ productId, zoneId, currency, addonFeatures }) => {
+
+        component.name = null;
+        component.icon = null;
+        component.description = null;
+        component.plans = null;
+        component.features = null;
+
```

Comment:
At some point, we need to reset the `error` property too
`component.error = false;`

Should we do that here or at line 28?

Reply:
Thanks for the feedback, I added the error reset but in the context subscribe (not the value subscribe).